deep_quoridor/src/train_alphazero.py

```diff
@@ -46,6 +47,12 @@ def train_alphazero(
         # members, so we pass in a GameParams object. We have to call start_game
         # because it calls the plugin's internal _intialize method which sets up metrics.
         wandb_train_plugin.start_game(game=args, agent1=training_agent, agent2=training_agent)
+        training_agent.set_wandb_run(wandb_train_plugin.run)
+
+        # Compute the tournament metrics with the initial model, possibly random initialized, to
+        # be able to see how it evolves from there
+        wandb_train_plugin.episode_count = 0
+        wandb_train_plugin.compute_tournament_metrics(str(current_filename))
```

Comment:
The untrained model can already beat some agents during MCTS, so it's good to have it as a base